src/components/base-address/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/base-address",
-  "version": "2.0.9",
+  "version": "2.0.10",
```

Comment:
Don't update this -- the publish step (done after your PR is merged) handles this.

Also, we'll want to change the major version number!